.github/workflow-templates/browser-testing.yml

```
@@ -0,0 +1,63 @@
+name: Browser Testing
+
+on: [push]
+
+jobs:
+  run:
+    name: Browser
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        browser:
+          - chrome
+          - firefox
+        os:
+          - ubuntu-latest
+          - windows-latest
+
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+
+      - name: Install Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14
+
+      - name: Get npm cache directory
+        id: npm-cache-dir
+        run: |
+          echo "::set-output name=dir::$(npm config get cache)"
+
+      - uses: actions/cache@v2
```

This may be breaking, not sure
```suggestion
      - uses: actions/cache@v3
```